lib/node/rpc.js

```diff
@@ -719,6 +724,63 @@ class RPC extends RPCBase {
     return await this.headerToJSON(entry);
   }
 
+  async getCFilter(args, help) {
+    if (help || args.length < 1 || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'getcfilter "hash" type');
+
+    if (!this.node.bip157)
+      throw new RPCError(errs.MISC_ERROR, 'Compact filter indexing disabled');
```

Comment:
If I understand correctly, `--bip157` is the argument to enable SERVING filters to light clients -- but shouldn't this command still be available as long as `--index-filters=true` ? The filters are on my disk, I should be able to get 'em.

Reply:
Hmm, maybe we should stick to one flag to avoid confusion? Do we need `--index-filters` and `--bip157`?

Reply:
nevermind, fixed.